src/main/scala/vectorpipe/osm/Element.scala

```diff
@@ -123,10 +124,11 @@ case class Member(
   ref: Long,
   role: String)
 
-case class ElementData(meta: ElementMeta, tagMap: Map[String, String])
+/** All Elements have common attributes, and may also have additional "tags". */
+@Lenses case class ElementData(meta: ElementMeta, tagMap: Map[String, String])
 
 /** All Element types have these attributes in common. */
-case class ElementMeta(
+@Lenses case class ElementMeta(
```

Comment:
I prefer this on a separate line, but I don't think this preference rises to the level of a standard

Reply:
The Monocle author does it that way in his docs: https://julien-truffaut.github.io/Monocle/optics/lens.html